mkdocs/docs/getting-started/install.md

```diff
@@ -26,7 +26,7 @@ hide:
     Pre-compiled Iter8 binaries for many platforms are available [here](https://github.com/iter8-tools/iter8/releases). Uncompress the iter8-X-Y.tar.gz archive for your platform, and move the `iter8` binary to any folder in your PATH.
 
 === "Source"
-    Build Iter8 from source as follows. Go `1.16+` is a pre-requisite.
+    Build Iter8 from source as follows. Go `1.17+` is a pre-requisite.
```

Comment:
What is our dependency on go 1.17?

Reply:
sigs.k8s.io/yaml v1.3.0 -- seems to complain for go 1.16. Note that for an end-user using brew (there is probably no go dependency; MacOS binary will be installed). 

For an enduser installing Iter8 using binaries, there is no go dependency.

For developers building from source, or for end-users using `go install`, go1.17 is required.